xtask/src/docgen/extract.rs

```diff
@@ -204,6 +204,9 @@ impl Parse for RuleTests {
 
         ok_examples.truncate(30);
 
+        // If there is a trailing comma, consumes it.
+        let _ = input.parse::<Token!(,)>();
+
```

Comment:
This change is related to #5, required to run `cargo xtask docgen` on `rule_test!` containing trailing comma.